include/industry_standard/spdm.h

```diff
@@ -560,6 +562,9 @@ typedef struct {
 /* Maximum size, in bytes, of a certificate chain. */
 #define SPDM_MAX_CERTIFICATE_CHAIN_SIZE 65535
 
+/* Maximum size, in bytes, of a measurement extension log. 0xFFFFFFFF - 1 - 16 = 4,294,967,279 */
+#define SPDM_MAX_MEASUREMENT_EXTENSION_LOG_SIZE 4294967279
```

Comment:
This should be for the `PortionLength` field and so should be `0xFFFFFFFF`. We may have another `#define` for maximum size of  DMTF Measurement Extension Log Format.

Reply:
Thanks. I have fixed the value.